docs/products/flink/overview.rst

```diff
@@ -0,0 +1,97 @@
+Service overview
+================
+
+This article gives you an overview of the service plans and features available in Aiven for Apache Flink beta.
```

Comment:
Should this be highlighted in a Note or Warning... to make clear this is a temporary reference?

Reply:
I'd leave this as is - essentially this topic would remain in place later on as well, but the content would be updated for the GA release, in which case the "beta" reference would also be dropped ;)